README.md

```diff
@@ -118,7 +118,7 @@ Put the following into your `~/.emacs`:
 ;; Teach compile the syntax of the kibit output
 (require 'compile)
 (add-to-list 'compilation-error-regexp-alist-alist
-	     '(kibit "At \\([^:]+\\):\\([[:digit:]]+\\):" 1 2 nil 0))
+	     '(kibit "At \\([^:]+\\):\\([[:digit:]]+\\):" 1 2 nil 2))
```

Comment:
Yes, that is my understanding, anyway. Next-error seems to ignore the info level messages.